code/src/Components/ThirdQuestion.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+const ThirdQuestion = ({
+    where, 
+    onWhereChange,
+    onStepChange,
+
+}) => {
+    return (
+        <form>
+            <label>
+                <input 
+                type="radio" 
+                value="Hemma hos mig"
+                onChange={() => onWhereChange('Hemma hos mig')}
+                checked={where === 'Hemma hos mig'}
+            />
+             </label>
+
+              <label>
+                <input 
+                type="radio" 
+                value="Hemma hos någon annan"
+                onChange={() => onWhereChange('Hemma hos någon annan')}
+                checked={where === 'Hemma hos någon annan'}
+            />
+               </label>
+               <label>
+                <input 
+                type="radio" 
+                value="På restaurang"
+                onChange={() => onWhereChange('På restaurang')}
+                checked={where === 'På restaurang'}
+           />
+              </label>
```

Comment:
This is nice way to display the radio. Or you can store the value to an array and the use map method to map over it, so we can keep the code cleaner, and be able to change the data faster